### PR TITLE
ddl/ingest: add failpoint to change flush local interval

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve?

Problem Summary:

This PR adds a failpoint to change flush local interval, thus we can change this in test environment.

### What changed and how does it work?

See code changes.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]
maybe add the unit "ms" to the failpoint name

File: pkg/ddl/ingest/backend.go [hunk 1]
```diff
@@ -269,8 +269,14 @@ func (bc *litBackendCtx) ShouldSync(mode FlushMode) (shouldFlush bool, shouldImp
 	if mode == FlushModeForceLocalAndCheckDiskQuota {
 		shouldFlush = true
 	} else {
+		interval := bc.updateInterval
+		failpoint.Inject("mockSyncInterval", func(val failpoint.Value) {
```
Comment (okJiang) [inline pkg/ddl/ingest/backend.go:273]: where is `mockSyncInterval`?

Comment (tangenta) [reply pkg/ddl/ingest/backend.go]: Do you mean the way to enable `mockSyncInterval`?

First, we need to set the environment variable `GO_FAILPOINTS` to `github.com/pingcap/tidb/pkg/server/enableTestAPI=return` before startup, in order to turn on the ability to toggle failpoint dynamically through HTTP.

Second, send a PUT request to status port to change `mockSyncInterval`:

```
curl -X PUT http://127.0.0.1:10080/fail/github.com/pingcap/tidb/pkg/ddl/ingest/mockSyncInterval -d "return(3000)"
```

**Pull request merged**